tests/test_trustme.py

```diff
@@ -359,3 +361,44 @@ def test_backcompat():
     ca = CA()
     # We can still use the old name
     ca.issue_server_cert(u"example.com")
+
+
+def test_CN():
+    ca = CA()
```

Comment:
I'd put this to fixture so that it'd be possible to put different tests into different functions more easily.